### ISSUE TITLE
Add VoteIT? - open source tool for online and transparent meetings

### ISSUE THREAD

**Issue by mattiasaxell** [okfse/opengovse#16]
@peterk what do you think about adding VoteIT to opengov.se? - a tool for transparent, digital decision-making meetings for boards etc? http://www.voteit.se/om-voteit/ I guess it can be interesting? 
Source code: https://github.com/VoteIT

@robinharms VoteIT can be used in public sector as well right?

**Comment by robinharms** [comment]
It's licensed under GPLv2 so you can use it for whatever you want :D

But it's a good idea to go through the VoteIT organisation and get some
training, since the UI refactoring isn't done. In other words: The admin UI
leaves a lot to be desired. It's easy for participants though.

Cheers,
/r

2016-01-27 11:24 GMT+01:00 Mattias Axell notifications@github.com:

> @peterk https://github.com/peterk what do you think about adding VoteIT
> to opengovse? - a tool for transparent, digital decision-making meetings
> for boards etc? http://wwwvoteitse/om-voteit/ I guess it can be
> interesting?
> Source code: https://githubcom/VoteIT
> 
> @robinharms https://github.com/robinharms VoteIT can be used in public
> sector as well right?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/okfse/opengovse/issues/16.

## 

Betahaus
phone: +46 70-333 00 10
web: http://www.betahaus.net

**Comment by mattiasaxell** [comment]
@robinharms In that sense, yes, great! Are there any examples or case you know of public sector using it? 

If tool is usable and relevant in line with open gov then it would be great to have
- a description in Swedish including that guideline 
- examples of meetings or processes in public sector where it would be suitable

**Comment by robinharms** [comment]
Hi, It's all @ www.voteit.se

There are about 100 organisations using it each year so yeah, there are a few examples to choose from :D

**Comment by peterk** [comment]
Maybe this is a good case for government transparency if there is a government agency/municipality using it, but I can't find info about that on the webpage?

**Comment by mattiasaxell** [comment]
Yes, agree. @robinharms is it possible that you can name some examples in public sector using VoteIT? I can't find info of public sector organisations using it at www.voteit.se, only member organisations.

**Comment by robinharms** [comment]
I wish, but there aren't any good examples as far as i know. So far, it's almost exclusively NGOs. There's even a bigger interest from regular commercial companies to use it. I guess that also says a lot about the state of how the public sector views interaction with citizens. :/

The only example i can think of straight away is MUCF using it to plan parts of a conference with VoteIT. Participants made proposals on what the government representatives should be asked during a session. So not exactly a breath-taking level of democratic engagement :)

In fact, if you know of any examples of public sector (in Sweden) engaging in more open-style decision making I'd love to know.

When it comes to other NOGs using it, only the members are published on voteit.se

**Comment by robinharms** [comment]
But back to the initial question: Please do have it on the site if you wish. But I'm afraid it can't serve as an example of government transparency or democratic processes outside of elections :)

**Comment by mattiasaxell** [comment]
@robinharms OK. That is an interesting issue but different question to explore and research.

If it can't serve as an example (yet) in opengov I guess we should not put it there now but later if it has become relevant and we have confirmed cases and public sector organizations using it.

**Comment by mattiasaxell** [comment]
I think I found two interesting international tools: 
http://democracyos.org/
and http://pol.is/
- What do you think @peterk - interesting? International usecases of Argentina & Taiwan however not used in Sweden but open source and possible to implement in Sweden. And regarding all additions let's post those on social media. :)

**Comment by peterk** [comment]
Yes! Maybe international use cases could lower the barrier for swedish public sector orgs who want to try it.

**Comment by mattiasaxell** [comment]
Closing as it is available on https://civictech.se/projects/
